debian/base/Dockerfile.rpm

```diff
@@ -70,10 +70,10 @@ RUN echo "===> Installing curl wget netcat python...." \
                 git \
                 wget \
                 nc \
-                python
+                python2
```

Comment:
What specifically needs python2?

Reply:
It's not the version of python that changes.
It's the name of the package.

Reply:
I understand the package changed. I was curious what needed Python2 that has not already been ported.

Reply:
I think there are a lot of things out of date here. 

For now the objective is to hopefully have Confluent roll forward the ancient OS's without breaking too many things.

 I think removing Python should be done separately from this, if indeed there are no more deps.